### ISSUE TITLE
Disabling horizontal scrolling

### ISSUE THREAD

**Issue by caliber01** [KinToday/kin-web-client#23]
Good evening, I have a very sensitive touchpad and whenever I scroll the calendar vertically in the week view I end up accidentally scrolling horizontally as well, which changes the current week. It makes the calendar completely unusable for me. Could you please include an option to disable the horizontal scrolling in the settings?

**Comment by zowesiouff** [comment]
> very sensitive touchpad

Out of curiosity, may I ask what's your touchpad / settings?

In theory our wheel normalizer is pretty good to accommodate as many setups as possible. Before introducing new settings (which is something I'm reluctant about: the whole simplicity / defaults should be right the first time debate), I will try to get a page setup for us to test your touchpad if that's alright with you? (and gather some data: number of scroll events, offsets generated ... ) Hopefully we can accommodate your setup while playing with some of the "magic numbers" used to wiggle the scroll trigger:
https://github.com/KinToday/kin-web-client/blob/master/src/client/calendar.js#L253

**Comment by caliber01** [comment]
@zowesiouff Yes, I am absolutely willing to participate. I am using Gnome and here are my touchpad settings. 
![screenshot from 2017-04-14 12-08-28](https://cloud.githubusercontent.com/assets/5463414/25039238/50bf503a-210b-11e7-8b86-1779cf78426d.png)

**Comment by adambyrtek** [comment]
I experience the same problem with Chrome on Linux as well.
